README.md

```diff
@@ -31,6 +31,19 @@ $ export AUTH_USERNAME="vault"
 $ export AUTH_PASSWORD="broker-secret-password"
 ```
 
+### Creating a Dev Environment
+
+Note: This section takes a good amount of time (likely, an hour or more) due to downloading a 16gb ISO and VirtualBox taking a while to create an environment.
+
+1. Register for an account at `https://pivotal.io`.
+2. Install the `cf` CLI: https://docs.cloudfoundry.org/cf-cli/install-go-cli.html.
+3. Ensure you have VirtualBox and Java 7 installed.
+4. Ensure you have ~30gb of disk storage available in your local environment for the VirtualBox ISO and env `cf` will create.
+5. Go through installation and configuration of `cf` thru the page on the `cf login` step here: https://pivotal.io/platform/pcf-tutorials/getting-started-with-pivotal-cloud-foundry-dev/introduction. Ignore the sample app steps regarding a Spring app.
+6. Note that when logging in following the example, the username is literally `user` and the password is literally `pass`, not the username and password you created during registration.
+7. Run Vault locally: `$ vault server -dev`. Use the Vault token returned as the `VAULT_TOKEN` value exported above.
+8. Make Vault reachable to our container using ngrok (https://ngrok.com/): `$ ngrok http 8200`. Use the http URL shown as the `VAULT_ADDR` value exported above (example value: "http://4d4053a3.ngrok.io").
```

Comment:
You say "container", but the preceding instructions make me think you meant "vm"...

Reply:
Ah I did that because later the terminal output starts calling it a container! Yes it will make no sense to the user calling it that at this point. Will update. Thank you!